projects/flop_count_example/Flop_Count_Example.ipynb

```diff
@@ -0,0 +1,341 @@
+    "\n",
+    "```\n",
+    "def my_counter_function(inputs: List[torch._C.Value], outputs: List[torch._C.Value]) -> float\n",
+    "```\n",
+    "\n",
+    "where objects of type `torch._C.Value` represent the symbolic inputs and outputs for the node in the graph (i.e. they're not actual concrete `Tensor`s). Check out the `fvcore` codebase for [examples of how to write counter functions](https://github.com/facebookresearch/fvcore/blob/master/fvcore/nn/jit_handles.py).\n",
+    "\n",
+    "### Customizing the Counter\n",
+    "\n",
+    "To register additional counter functions or override the counter's default implementation for specific operations, use the `counter_overrides` argument, which takes the form of a dictionary mapping operator names to the corresponding counter functions you wish to use. As an example, consider the following simple module:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "aa3b8c44",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class MyModule(nn.Module):\n",
+    "    def forward(self,x,y,z):\n",
+    "        return x + y * z"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fdb5c1db",
+   "metadata": {},
+   "source": [
+    "By default, the elementwise addition or multiplication of two tensors of shape `N x M` will contribute `N x M` flops to the total computational complexity of a model (since each scalar addition/multiplication is counted as 1 flop):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "66b78a26",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "True"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "N = 5\n",
+    "M = 32\n",
+    "\n",
+    "inp = torch.randn(N, M)\n",
+    "\n",
+    "flops,_,_ = count_flops(MyModule(), (inp, inp, inp))\n",
+    "flops == 2 * N * M"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ea4d5521",
+   "metadata": {},
+   "source": [
+    "However, let's say that you wanted to ignore all the flops coming from addition operations. You could do this as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "5f7a806c",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "True"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "new_flops, _, _ = count_flops(\n",
+    "    MyModule(), \n",
+    "    (inp, inp, inp), \n",
+    "    counter_overrides = {\n",
+    "        \"aten::add\": lambda inps,outs: 0.0\n",
+    "    }\n",
+    ")\n",
+    "\n",
+    "new_flops == N * M # only the flops from multiplications are counted"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8c05767f",
+   "metadata": {},
+   "outputs": [],
+   "source": []
```

Comment:
Should add some comment or description here summarizing the output of the previous cell and concluding.